style.css

```diff
@@ -276,6 +280,26 @@ nav .menu {
   color: #6070ff;
 }
 
+.see-project a:hover {
+  background-color: #6070ff;
+  color: #fff;
+}
+
+.see-project a:active {
+  background-color: #2230d2;
+  color: #fff;
+}
+
+.get-resume a:hover {
+  background-color: #6070ff;
+  color: #fff;
+}
+
+.get-resume a:active {
+  background-color: #2230d2;
+  color: #fff;
+}
```

Comment:
- [ ] Good job adding the styling for the `active` and `hover` states all that's left is the `disabled` state, please do that.  😁